awx/main/management/commands/run_cache_clear.py

```diff
@@ -18,7 +19,7 @@ class Command(BaseCommand):
 
     def handle(self, *arg, **options):
         try:
-            with pg_bus_conn(new_connection=True) as conn:
```

Comment:
Since these services are running separately won't they have establish new connections anyway? At least that is my understanding from my previous PR.